go.php

```diff
@@ -82,13 +82,49 @@
 
 header('HTTP/1.1 302 Moved Temporarily');
 if (array_key_exists($from, $mapping)) {
-    header('Location: ' . $location . $mapping[$from]);
+    $subPath = $mapping[$from];
 } else {
     if (strpos($from, 'admin-') === 0) {
-        header('Location: ' . $location . '/admin_manual');
+        $subPath = '/admin_manual';
     } else if (strpos($from, 'developer-') === 0) {
-        header('Location: ' . $location . '/developer_manual');
+        $subPath = '/developer_manual';
     } else {
-        header('Location: ' . $location . '/user_manual');
+        $subPath = '/user_manual';
     }
 }
+
+$subPathParts = explode('/', $subPath);
+$manual = $subPathParts[1];
```

Comment:
```suggestion
$manual = $subPathParts[1] ?? '';
```
Avoid logging PHP errors for bad URLs.